examples/timeline-charts-example/src/Dashboard.jsx

```diff
@@ -0,0 +1,202 @@
+import React, { useRef, useCallback, useState } from "react";
+
+import { withStyles } from "@material-ui/core/styles";
+import Slider from "@material-ui/core/Slider";
+import ChartsEmbedSDK from "@mongodb-js/charts-embed-dom";
+import { useEffect } from "react";
+
+const firstOlympicsYear = 1896;
+const lastOlympicsYear = 2016;
+const timelineInterval = 2000; //ms
+
+let timerId;
+let replay = false;
+
+const sdk = new ChartsEmbedSDK({
+  baseUrl: "https://charts.mongodb.com/charts-data-science-project-aygif", // Optional: ~REPLACE~ with the Base URL from your Embed Chart dialog
+});
+
+const columnChart = sdk.createChart({
+  chartId: "ff518bbb-923c-4c2c-91f5-4a2b3137f312", // Optional: ~REPLACE~ with the Chart ID from your Embed Chart dialog
+});
+
+const geoChart = sdk.createChart({
+  chartId: "b1983061-ee44-40ad-9c45-4bb1d4e74884", // Optional: ~REPLACE~ with the Chart ID from your Embed Chart dialog
+});
+
+const PrettoSlider = withStyles({
```

Comment:
typo: `PrettySlider`

Could consider changing the name to: TimelineSlider

Reply:
I'd also move this out into a separate file and just `import PrettySlider from './PrettySlider'`